blog.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Adopt a Dog</title>
+  </head>
+  <style media="screen">
+
+    header {
+      background: grey;
+      width: 100%;
+      height: 15%;
+    }
+
+    footer {
+      background: grey;
+      width: 100%;
+      height: 15%;
+    }
+
+    .cart {
+      width: 2rem;
+      position: absolute;
+      top: 5%;
+      right: 10%;
+      color: black;
+    }
+
+    .checkout-button {
+      width: 2rem;
+      position: absolute;
+      top: 10%;
+      right: 10%;
+      color: black;
+
+    }
+
+    .logo {
+      width: 30%;
+    }
+
+    .container {
+      display: flexbox;
+      width: 25%;
+      float: left;
+
+    }
+
+  </style>
+  <body>
+    <header>
+      <img src="images/adoptadog-logo.svg" class="logo" alt="Dog sitting next to AdoptaDog text">
+      <p>Seattle's premiere dog adoption agency</p>
+      <img src="images/cart-icon.svg" class="cart" alt="shopping cart">
+      <button class=checkout-button><a href="checkout.html">checkout</a></button>
+    </header>
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="dogs.html">Dogs</a></li>
+        <li><a href="blog.html">Blog</a></li>
+      </ul>
+    </nav>
+    <section>
+      <h1>Adopt a Blog</h1>
```

Comment:
I like how the blog posts are wrapped in a section, very smart. You'll want to wrap the  blog posts up with something (a div or maybe a better semantic HTML5 element....)